modules/API/pages/gsql-endpoints.adoc

```diff
@@ -4162,3 +4162,2356 @@ Response::
+
+==== Example
+
+[,tabs]
+====
+Request::
++
+--
+[source,bash]
+----
+curl -H 'Content-Type: application/json' -X POST 'http://localhost:14240/gsql/v1/loading-jobs/run?graph=g' 
+payload could be one of these:
+kafka loading: [{"name":"load_kafka","dataSources":[{"filename":"f1","name":"k1","path":"","config":{"topic":"regress7715","partition_list":[{"start_offset":-2,"partition":0}]}}],"streaming":false}]
+s3: [{"name":"load_comment","streaming":true,"dataSources":[{"filename":"file_Comment","name":"s1","path":"s3-loading-test/tg_ldbc_snb/sf0.1_csv/dynamic/Comment"}]}]
+file loading: [{"name":"load_job","sys.data_root":"/tmp","dataSources":[{"filename":"f","path":"./data","name":"file"}]}]
+full parameters: [{"name":"load_job","sys.data_root":"/tmp","verbose":true,"dryrun":true,"interval":1,"maxNumError":1,"maxPercentError":1,"dataSources":[{"filename":"f","path":"./data","name":"file"}]}]
+----
+--
+Response::
++
+--
+[source.wrap,console]
+----
+{"error": false,"message":"Successfully ran loading job(s): [jobName]", "jobIds": ["jobId"]}
+----
+--
+====
+===== Parameters:
+
+//[width="100%",cols="25%,25%,50%",options="header",]
+[%autowidth]
+|===
+| Name | Required | Description
+
+| `graph`
+| yes
+| run the loading job under which graph
+|===
+
+'''
+
+=== drop a loading job
+
+`DELETE /gsql/v1/loading-jobs/{jobName}?graph=[graphName]`
+
+This endpoint is used to drop a loading job by name.
+
+==== Example
+
+[,tabs]
+====
+Request::
++
+--
+[source,bash]
+----
+curl -H 'Content-Type: application/json' -X DELETE 'http://localhost:14240/gsql/v1/loading-jobs/jobName?graph=g' 
+----
+--
+Response::
++
+--
+[source.wrap,console]
+----
+{"error": false,"message":"Successfully drop loading job 'jobName'."}
+----
+--
+====
+===== Parameters:
+
+//[width="100%",cols="25%,25%,50%",options="header",]
+[%autowidth]
+|===
+| Name | Required | Description
+
+| `graph`
+| yes
+| drop the loading job under which graph
+|===
+
+'''
+
+=== abort loading job
+
+`GET /gsql/v1/loading-jobs/abort?graph=[graphName]&jobIds=[job1,job2,...]&isPause=[true/false]`
+
+This endpoint is used to abort mutilple running loading jobs.
+
+==== Example
+
+[,tabs]
+====
+Request::
++
+--
+[source,bash]
+----
+curl -H 'Content-Type: application/json' -X GET 'http://localhost:14240/gsql/v1/loading-jobs?graph=g&jobIds=jobId1' 
+curl -H 'Content-Type: application/json' -X GET 'http://localhost:14240/gsql/v1/loading-jobs?graph=g&jobIds=jobId1&isPause=true' 
+----
+--
+Response::
++
+--
+[source.wrap,console]
+----
+default behavior: {"error": false,"message":"Successfully dropped loading job(s): [jobId1]."}
+{"error": false,"message":"Successfully paused loading job(s): [jobId1]."}
+----
+--
+====
+===== Parameters:
+
+//[width="100%",cols="25%,25%,50%",options="header",]
+[%autowidth]
+|===
+| Name | Required | Description
+
+| `graph`
+| yes
+| run the loading job under which graph
+
+| `jobId`
+| yes
+| the loading job Id
+
+| `isPause`
+| no
+| pause or abort the loading job

```

Comment:
ambiguous, `true means pause or abort the loading job, false means run the loading job.`